### PR TITLE
Vendor in latest containers/common

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

#### Does this PR introduce a user-facing change?

```release-note
Network Lock file moved from network config directories to /run directory.
```

### CODE REVIEW

File: test/e2e/network_test.go [hunk 1]
```diff
@@ -38,26 +38,6 @@ var _ = Describe("Podman network", func() {
...
-		// default network always exists
-		Expect(session.OutputToStringArray()).To(HaveLen(1))
-
-		// check that the only file in the directory is the network lockfile
-		dir, err := os.Open(netDir)
-		Expect(err).ToNot(HaveOccurred())
-		names, err := dir.Readdirnames(5)
-		Expect(err).ToNot(HaveOccurred())
-		Expect(names).To(HaveLen(1))
-		Expect(names[0]).To(Or(Equal("netavark.lock"), Equal("cni.lock")))
```
Comment (Luap99) [inline test/e2e/network_test.go:58]: keep the test, just remove that part here since it is no longer applicable

**Comment by rhatdan** [comment]
@Luap99 @vrothberg @giuseppe @flouthoc PTAL
@containers/podman-maintainers PTAL

**Comment by umohnani8** [comment]
LGTM

**Review by flouthoc — APPROVED** [review]
LGTM
/lgtm

**Pull request merged**